data/en/ukis_0001.json

```
@@ -1,277 +1,147 @@
 {
-    "eq_id": "ukis",
+    "eq_id": "ukis_0001",
```

Hi Sam. I see why this has changed; normally you're not incorrect - the eq_id is of that format. However, to avoid unpredicted errors, if the existing eq_id is different within the schema when we go to change it we normally leave it alone. Just in case.

```suggestion
    "eq_id": "ukis",
```